app/src/main/java/net/kuama/documentscanner/viewmodels/CropperModel.kt

```diff
@@ -32,9 +30,8 @@ class CropperModel : ViewModel() {
                 )
             ) { either ->
                 either.fold(::handleFailure) { preview ->
-                    analyze(preview, returnOriginalMat = true) { pair ->
```

Comment:
without value, what is the default? here it seems true, in the signature of analyze false..
it is important to decide what is the correct behavior now in the usecase